server/events/terraform/terraform_client.go

```diff
@@ -261,6 +261,8 @@ func (c *DefaultClient) prepCmd(log *logging.SimpleLogger, v *version.Version, w
 	// We add custom variables so that if `extra_args` is specified with env
 	// vars then they'll be substituted.
 	envVars := []string{
+		// Pass user-agent for Terraform and provider calls.
+		"TF_APPEND_USER_AGENT=Atlantis/1.0",
```

Comment:
anyway we can use the actual atlantis version here? I'm in favor of the overall change.